eregs_core/static/regulations/js/source/models/meta-model.js

```diff
@@ -120,7 +120,12 @@ var MetaModel = Backbone.Model.extend({
         }
 
         if (typeof this.supplementalPath !== 'undefined') {
-            url += this.supplementalPath + '/';
+            if (this.supplementalPath == 'sidebar') {
+                url = '/sidebar_partial/'
+            }
+            else {
+                url += this.supplementalPath + '/';
```

Comment:
I guess I was thinking of renaming them all to `<function>_partial` or something like that, but maybe it's just easier to change the backend scheme? I guess it doesn't really matter which side changes; on the backend you only have to modify `urls.py` so perhaps it's better to just do that?